app/src/main/java/mok/it/app/mokapp/feature/project_add_participants/AddOrRemoveMembersFragment.kt

```diff
@@ -0,0 +1,217 @@
+        val chipState =
+            rememberChipTextFieldState<Chip>() // this is not saved at configuration change yet
+        var searchQuery by rememberSaveable { mutableStateOf("") }
+        val filteredUnselectedUsers = getFilteredUsers(
+            searchQuery,
+            chipState,
+            uiState.selectedUserIds
+        ).filter { !uiState.selectedUserIds.contains(it.documentId) }
+        val users = viewModel.users.observeAsState(initial = emptyList()).value
+
+        var showDialog by rememberSaveable { mutableStateOf(DialogType.NONE) }
+
+        Scaffold(
+            bottomBar = {
+                Button(modifier = Modifier
+                    .padding(8.dp)
+                    .fillMaxWidth(),
+                    enabled = uiState.selectedUsersChanged,
+                    onClick = {
+                        showDialog = DialogType.CONFIRM
+                    }) {
+                    Text(text = "Mentés")
+                }
+            },
+        ) { padding ->
+            Column(
+                modifier = Modifier.padding(bottom = padding.calculateBottomPadding())
+            ) {
+                when (showDialog) {
+                    DialogType.CONFIRM -> {
+                        OkCancelDialog(
+                            text = "Biztos, hogy el szeretnéd menteni a módosításokat? Ha valakit törölsz a projektből, az elveszíti az összes eddig megszerzett mancsát rajta.",
+                            onConfirm = {
+                                viewModel.updateMembersOfProject()
+                                Toast.makeText(
+                                    context,
+                                    "Résztvevők listája módosítva!",
+                                    Toast.LENGTH_SHORT
+                                )
+                                    .show()
+                                showDialog = DialogType.NONE
+                            },
+                            onDismiss = {
+                                showDialog = DialogType.NONE
+                            },
+                        )
+                    }
+
+                    DialogType.NONE -> {}
+                }
+                SearchField(
+                    searchQuery = searchQuery,
+                    chipState = chipState,
+                    onValueChange = { searchQuery = it },
+                )
+
+                LazyColumn {// selected users
+                    items(users.filter { uiState.selectedUserIds.contains(it.documentId) }) { user ->
+                        UserRow(
+                            user,
+                            true,
+                        ) { _ ->
+                            viewModel.userSelectionClicked(user)
+                        }
+                    }
+                }
+                HorizontalDivider(modifier = Modifier.padding(8.dp))
+                if (filteredUnselectedUsers.isNotEmpty()) {
+                    LazyColumn {// all users
```

Comment:
ez a komment kicsit indokolatlan, hiszen ezek azok a userek akik még nem tagok, és azon belül filterelve is vannak